docs/index.md

```diff
@@ -43,7 +43,7 @@ exceptions and provide input and result objects.
 
 Read more about [how do use asynchronous features](./features/async.md).
 
-## Quick example
+## Installation and usage
```

Comment:
What's about moving this section before "How is it async first?"

Reply:
Sure